php/class-photonfill.php

```diff
@@ -102,9 +102,9 @@ public function setup() {
 			// Set our breakpoints.
 			/**
 			 * A breakpoint can accept the following parameters
```

Comment:
This looks like a comment that's better suited to the `photonfill_breakpoints` filter below. Both of these filters should have docblocks that conform to the WP standards for action hooks and filters:

https://make.wordpress.org/core/handbook/best-practices/inline-documentation-standards/php/#4-hooks-actions-and-filters

This information could therefore be expressed as part of the specification for the `$breakpoints` parameter in the filter, similar to this:

```php
<?php
/**
 * Allows for the breakpoint sizes and properties to be filtered.
 *
 * @param array $breakpoints {
 *     Key-value pairs describing breakpoints, where the key is the name of the 
 *     breakpoint, and the value is an array of properties for the breakpoint.
 *
 *     @type array $breakpoint_name {
 *         Properties for the breakpoint.
 *
 *         @type int $max Max width of element.
 *         @type int $min Min width of element.
 *         @type string $unit [px(default),em] Currently does not support vw unit, multi units or calc function.
 *     }
 * }
 */
```